docs/section12/Rule12-2.md

```diff
@@ -1,36 +1,40 @@
-
-# Receptacle - Rule 12-2
-
+# Receptacles - Rule 12-2
 **Rule ID:** 12-2  
-**Rule Description:** Number of spaces modeled in User RMR and Proposed RMR are the same  
-**Rule Assertion:** Proposed RMR = User RMR  
+**Rule Description:** Based on the space type, receptacle controls may be required by Section 8.4.2. Receptacle schedules shall be identical to the proposed design except when receptacle controls, installed in spaces where not required by Section 8.4.2, are included in the proposed building design.  
+**Rule Assertion:** Baseline RMD = Proposed RMD  
 **Appendix G Section:** Section Table G3.1-12 Modeling Requirements for the Proposed design  
 **Appendix G Section Reference:** None  
-
-**Applicability:** All required data elements exist for P_RMR and U_RMR  
-**Applicability Checks:** None  
-**Manual Check:** None  
-**Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
+**Evaluation Context:** Each miscellaneous equipment load
 
-## Rule Logic:  
-
-- Get the total number of spaces in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
+**Applicability Checks:** None
 
-  - For each thermal_block in building segment: ```thermal_block in building_segment.thermal_blocks:```
+**Function Call**
+match_data_element, compare_schedules
 
-  - For each zone in thermal block: ```zone_user in thermal_block_user.zones:```
 
-  - For each space in zone, add to the total number of spaces in the User model: ```for space_user in thermal_zone_user.spaces: num_of_spaces_user += 1```  
-
-- Get the total number of spaces in the building segment in the Proposed model: ```for building_segment_proposed in P_RMR.building.building_segments:```  
-
-  - For each thermal_block in building segment: ```thermal_block_proposed in building_segment_proposed.thermal_blocks:```
-
-  - For each zone in thermal block: ```zone_proposed in thermal_block_proposed.zones:```
-
-  - For each space in zone, add to the total number of spaces in the Proposed model: ```for space_proposed in zone_proposed.spaces: num_of_spaces_proposed += 1```  
+## Rule Logic:  
+- Create a list of the lighting space types that correspond to space types that may have receptacle control requirements in Section 8.4.1: `EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES = ["OFFICE_ENCLOSED", "CONFERENCE_MEETING_MULTIPURPOSE_ROOM", "COPY_PRINT_ROOM", "LOUNGE_BREAKROOM_HEALTH_CARE_FACILITY", "LOUNGE_BREAKROOM_ALL_OTHERS", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_PENITENTIARY", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_SCHOOL", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_ALL_OTHER", "OFFICE_OPEN_PLAN"]`  
+- For each space in the baseline RMD: `for space_b in B_RMD.spaces:`  
+  - Get the lighting space type: `space_type_b = space_b.space_type`
```

Comment:
should this be .lighting_space_type

Also, suggest adding a handler for if this is not defined.